client/codegen.yml

```diff
@@ -1,5 +1,9 @@
 overwrite: true
-schema: ["https://demo-open.msupply.org/graphql", "./packages/common/src/additions.schema.graphql"]
+schema:
+  [
+    'http://localhost:8000/graphql',
+    './packages/common/src/additions.schema.graphql',
+  ]
```

Comment:
Leave this in local while we're on the feature branch, you think?

Reply:
I suspect that we'll forget to change it back. Probably reasonable to keep it local though, and not rely on the demo server anyway.